AsyncDisplayKitTests/ASTextNodeSnapshotTests.m

```diff
@@ -32,4 +32,26 @@ - (void)testTextContainerInset
   ASSnapshotVerifyNode(textNode, nil);
 }
 
+- (void)testTextContainerInsetHighlight
+{
+  UIView *backgroundView = [[UIView alloc] initWithFrame:CGRectZero];
+  backgroundView.layer.as_allowsHighlightDrawing = YES;
+
+  ASTextNode *textNode = [[ASTextNode alloc] init];
+  textNode.attributedText = [[NSAttributedString alloc] initWithString:@"yolo"
+                                                            attributes:@{ NSFontAttributeName : [UIFont systemFontOfSize:30] }];
+
+  [textNode measureWithSizeRange:ASSizeRangeMake(CGSizeZero, CGSizeMake(CGFLOAT_MAX, CGFLOAT_MAX))];
+  textNode.frame = CGRectMake(50, 50, textNode.calculatedSize.width, textNode.calculatedSize.height);
+  textNode.textContainerInset = UIEdgeInsetsMake(5, 10, 10, 5);
+
+  [backgroundView addSubview:textNode.view];
+  backgroundView.frame = UIEdgeInsetsInsetRect(textNode.bounds, UIEdgeInsetsMake(-50, -50, -50, -50));
+
+  textNode.highlightRange = NSMakeRange(0, textNode.attributedText.length);
+
+  [ASSnapshotTestCase hackilySynchronouslyRecursivelyRenderNode:textNode];
+  FBSnapshotVerifyLayer(backgroundView.layer, nil);
```

Comment:
Awesome! Can these two lines be replaced by `ASSnapshotVerifyNode(textNode, nil)`?